internal/test/wpt/Makefile

```
@@ -1,3 +1,8 @@
 .PHONY: test
 test:
 	GOEXPERIMENT=jsonv2 go run . dom/nodes/Element-childElement dom/nodes/Document-getElementById
+
+# WIP target is used to run tests for currently worked on feature
+.PHONY: wip
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)